Monika After Story/game/script-topics.rpy

```diff
@@ -8917,6 +8917,250 @@ label monika_hydration:
+            m 5ekbfu "Ehehe~ I love you so much, [player]."
+    return
+
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_earthlikeplanets",category=['misc'],prompt="Earthlike Planets",random=True))
+
+label monika_earthlikeplanets:
+    m 1lua "Have you ever wondered how lucky we all are?"
+    m 4esa "If you think about it, the possibilities of having an existing planet that is suitable for human life are quite sparse."
+    m 4eka "Most of the celestial bodies out there don't support any kind of human activity."
+    m 4eku "Some of them have little to no gravity, no oxygen, or even terrain to stand on."
+    m 1eku "Yet, here we are, on planet Earth."
+    m 1hub "Isn't that great? You should be feeling lucky right now just for being alive."
+    m 2euu "It's almost like life itself gave us the oportunity to live..."
+    m 3esb "You better remember that next time you are feeling down, [player]!"
+    m 3tsb "Or... You could also come to me and allow me to lift your mood~"
+    m 1hublb "Ehehe~"
+    return
+
+define persistent._mas_didplayerbetray = None
+
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_betrayal",category=['misc'],prompt="Betrayal",random=True))
+
+label monika_betrayal:
+    m 1eka "...Say, [player]?"
+    m 1lka "Something came to my mind just now."
+    m 1eka "Have you ever betrayed someone by any means?"
+    m 4eka "I don't mean to 'betray' little promises with friends that could even be considered jokes."
+    m 4eku "I mean it in a real way... You know, breaking the trust someone had with you because you did something terrible to them."
+    m "You don't have to answer if you don't want to. It's completely understandable.{nw}"
+    $ _history_list.pop()
+    menu:
+        m "You don't have to answer if you don't want to, it's completely understandable.{fast}"
+        "Yes, I have betrayed someone.":
+            $ persistent._mas_didplayerbetray = True
+            m 2ekc "..."
+            m "It's alright, [player]."
+            m 3ekc "I know you probably didn't mean to do that."
+            m 3ekb "We all mess up sometimes, don't we?"
+            m "Maybe you didn't do it on purpose."
+            m "Maybe the situation pushed you into doing it in some way."
+            m 3ekc "Whatever happened, you have to make sure to never do that again, alright?"
+            m "You have to promise me."
+            m "Trust is not something to play with."
+            m "Try to keep that in mind."
+
+        "No, I haven't betrayed anyone.":
+            $ persistent._mas_didplayerbetray = False
+            m 2huu "I'm so happy to hear that, [player]."
+            m 3efb "People's trust isn't something to be played with."
+            m "So I'm glad to hear that from you."
+            m 3esa "And just so you know..."
+            m 3hublb "I...{w=1}would never betray you, [player]~."
+
+        "...":
```

Comment:
This path should ideally have a `no_talk` player model var rather than just not setting anything as it returns derandom.